spec/models/provider_enrichment_spec.rb

```diff
@@ -22,14 +22,38 @@
   end
 
   describe 'auditing' do
-    it { should be_audited }
+    it { should be_audited.except(:json_data) }
 
     it 'can link to Provider' do
       provider = create(:provider)
       provider_enrichment = create(:provider_enrichment, provider: provider)
 
       expect(provider_enrichment.audits.last.associated).to eq(provider)
     end
+
+    context 'when dealing with a `jsonb_accessor` attribute' do
+      let(:accrediting_provider_enrichment) do
+        { "UcasProviderCode" => "ABC", "Description" => "A test value" }
+      end
+      let(:provider_enrichment) { create(:provider_enrichment) }
+
+      before do
+        provider_enrichment.accrediting_provider_enrichments = [accrediting_provider_enrichment]
+        provider_enrichment.save
+      end
+
+      it 'does not audit `json_data` attribute' do
+        expect(provider_enrichment.audits.last.audited_changes).to_not have_key('json_data')
+      end
+
+      it 'does include `accrediting_provider_enrichments` at top level of audited_changes' do
+        changes = provider_enrichment.audits.last.audited_changes
+        expect(changes['accrediting_provider_enrichments']).to include(
+          nil,
+          [include(accrediting_provider_enrichment)]
```

Comment:
Why is `include` in here? Should it be:

```suggestion
          accrediting_provider_enrichment
```

Reply:
`accrediting_provider_enrichments` is an array. And the Data stored to the DB also includes  `errors: {}, validation_context: nil`. So make it less brittle.

```ruby
expect(changes['accrediting_provider_enrichments']).to include(
  nil,
  accrediting_provider_enrichment
)

expected [nil, [{"Description" => "A test value", "UcasProviderCode" => "ABC"}]] to include {"UcasProviderCode" => "ABC", "Description" => "A test value"}
Diff:

 [
   nil,
-   {
-     "UcasProviderCode" => "ABC",
-     "Description" => "A test value"
-   }
+   [{ "Description" => "A test value", "UcasProviderCode" => "ABC" }]
 ]
````

Reply:
Cool. Didn't realise the nested `include` did that. Nice one. 👍